compatibility/bazel_tools/daml.cc.tpl

```diff
@@ -0,0 +1,87 @@
+#include "tools/cpp/runfiles/runfiles.h"
+#include <codecvt>
+#include <iostream>
+#include <locale>
+#include <memory>
+#include <unistd.h>
+#if defined(_WIN32)
+#include <windows.h>
+#endif
+
+using bazel::tools::cpp::runfiles::Runfiles;
+
+/*
+NOTE (MK): Why is this not just an sh_binary?
```

Comment:
Why not a `haskell_binary`?

Reply:
I got sufficiently scared of cross-platform process management that I wanted to know exactly what is going on here so this seemed like the easiest option. Now that it does work, a Haskell binary would probably do the trick but having a several megabyte binary for this makes me a bit sad.